digitaloceanmodule.php

```diff
@@ -331,6 +331,30 @@ public function getPackageFields($vars = null) {
         return $fields;
     }
 
+	public function getallImagesDropdown($package) {
+        $row = $this->getModuleRow($package->module_row);
+        $api = $this->getApi($row->meta->apiKey);
+        $result = $api->getlongGetResults("images");
+        $dp = array();
+		foreach ($result->images as $key => $value)
+		{
+		$dp[$result->images[$key]->slug] = $result->images[$key]->slug;
+		}
+		return $dp;
+    }
+	
+	public function getAllRegionsDropdown($package) {
+        $row = $this->getModuleRow($package->module_row);
```

Comment:
Indentation looks off here. You should follow the same syntax as the rest of the file.